src/components/molecules/PollBox/PollBox.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useCallback, useMemo } from "react";
+import { useForm, useFieldArray, Controller } from "react-hook-form";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPaperPlane, faPlus } from "@fortawesome/free-solid-svg-icons";
+import { InputField } from "components/atoms/InputField";
+import { MAX_QUESTIONS_NUMBER } from "settings";
+import "./PollBox.scss";
+
+export type Question = {
+  name: string;
+};
+
+export type PollValues = {
+  topic: string;
+  questions: Question[];
+};
+
+export interface PollBoxProps {
+  onSubmit: (args: PollValues) => void;
+}
+
+const defaultQuestion: Question = { name: "" };
+const defaultValues = {
+  topic: "",
+  questions: [defaultQuestion, defaultQuestion],
+};
+
+export const PollBox: React.FC<PollBoxProps> = ({ onSubmit }) => {
+  const { control, handleSubmit, reset, watch } = useForm<PollValues>({
+    defaultValues,
+  });
+  const { fields, append } = useFieldArray({ name: "questions", control });
+
+  const onCustomSubmit = handleSubmit((data) => {
+    onSubmit(data);
+    reset();
+  });
+
+  const isDisabled = useMemo(() => {
+    const [question1, question2] = watch("questions");
+    return !(watch("topic") && question1.name && question2.name);
+  }, [watch]);
+
+  const addChoice = useCallback(() => append(defaultQuestion), [append]);
+  const showAppend = useCallback(
+    (index) =>
+      index + 1 === fields.length && MAX_QUESTIONS_NUMBER > fields.length,
+    [fields]
+  );
+  const formatPlaceholder = useCallback(
+    (index) =>
+      index === 0
+        ? `Choice ${index + 1} (Max ${MAX_QUESTIONS_NUMBER} choices)`
+        : `Choice ${index + 1}`,
+    []
+  );
```

Comment:
I don't think this placeholder text is a good thing to include at all personally (as called out in the linked issue)

From memory, the designs only show it added to the first textbox, which just seems arbitrary and unbalanced. Either it should be added to all of them (at which point it'll likely feel overpowering and cluttered), or none of them (my personal preference).

@mcflyok @mike-lvov Thoughts?

Reply:
All of this `+ 1` logic here + in `showAppend`/etc should happen within the `.map` of `renderChoiceFields` rather than spread out into all of these helper functions IMO. Then instead of `index`, it should just be `choiceNumber` or similar.

Reply:
@0xdevalias I'm not sure we are on the same page. What's your idea to upgrade `formatPlaceholder`?

Reply:
My suggestion is as outlined here: https://github.com/sparkletown/internal-sparkle-issues/issues/654#issuecomment-827448966

But that goes against what @mobilevinay / @sparkletown/product has said, even though I think that is a bad choice.

Thus me asking for @mike-lvov 's opinion here as well.

Reply:
@yurii-lubynets @0xdevalias
 IMO placeholder looks ugly. Let's remove that. My personal call is to remove the placeholder & not implementing the `2/8` counter as well. Let's keep the first version easy. If people find it confusing, we can add that later.

Reply:
https://github.com/sparkletown/internal-sparkle-issues/issues/654#issuecomment-831473257